### PR TITLE
[WNMGDS-1829] hgov logo and footer documentation

### PR DESCRIPTION
**Pull request by scheul93**
## Summary

* Adding doc page for Healthcare Logo  [original page](https://cmsgov.github.io/design-system/healthcare/components/logo/)
* Adding doc page for Healthcare Footer [original page](https://cmsgov.github.io/design-system/healthcare/patterns/footer/)
* Updating Gatsby config so that props can be pulled for child DS components

## How to test

`yarn start:gatsby`

Logo page:

Screen Shot 2022-08-11 at 10 03 01 AM

Footer page:
Screen Shot 2022-08-11 at 10 02 48 AM

### CODE REVIEW

File: packages/docs/content/components/footer/healthcare-footer.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <Alert variation="error">
+     <p class="ds-c-alert__text">
+        This component is only used for HealthCare. Please use the theme switcher to view the component with HealthCare styles.
+    </p>
+  </Alert>
+</ThemeContent>
+
+## Examples
+
+<StorybookExample
```
Comment (pwolfert) [inline packages/docs/content/components/footer/healthcare-footer.mdx:20]: The old footer page used a responsive example. Should we also provide a responsive example here? I guess they can always click through to storybook and do it there. I do like the minimalism of the normal examples.

File: packages/ds-healthcare-gov/src/styles/components/_Footer.scss [hunk 2]
```diff
@@ -40,15 +40,13 @@
...
-
     background-color: currentColor;
     border-radius: 50%;
     content: '';
     display: inline-block;
-    height: $hc-c-footer-list-marker-size;
+    height: $footer__list-marker-size;
     vertical-align: middle;
-    width: $hc-c-footer-list-marker-size;
+    width: $footer__list-marker-size;
```
Comment (pwolfert) [inline packages/ds-healthcare-gov/src/styles/components/_Footer.scss:49]: Nice 👌

File: packages/docs/content/components/footer/healthcare-footer.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: Healthcare.gov Footer
```
Comment (pwolfert) [inline packages/docs/content/components/footer/healthcare-footer.mdx:2]: I'm not sure why, but for some reason the sidebar is capitalizing `.Gov` 🤷‍♂️. Not a big deal, but pointing it out

**Review by pwolfert — APPROVED** [review]
LGTM 👍

File: packages/docs/content/components/footer/healthcare-footer.mdx (hunk 3 above)
Comment (scheul93) [reply packages/docs/content/components/footer/healthcare-footer.mdx:2]: Oh... I think the sidebar has a css text transform on it to capitalize text and that was leftover from when we were using filenames not frontmatter titles. Let me change that and see if we can remove the styling

**Review by forrestbaer — COMMENTED** [review]
My gatsby keeps crashing when I browse the site with various issues but all seem to be related to being unable to find 'temp query result's for various components.  Gonna see if there is anything I can do about it locally, neither of you ran into this? Try clicking around to a bunch of pages and see if it comes up?

**Comment by pwolfert** [comment]
> My gatsby keeps crashing...neither of you ran into this?

I haven't run into that error, no 😞

**Comment by scheul93** [comment]
I have not either. I'm on node `v16.15.1` & npm `8.11.0`

**Review by forrestbaer — APPROVED** [review]
Must be something local for me then, code & content looks good 👍🏻

**Pull request merged**